common/changes/@itwin/manage-versions-react/Pooja-show-included-changeset-in-named-versions-table_2023-11-06-12-10.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/manage-versions-react",
+      "comment": "Show included changesets in Named versions table on expand rows",
+      "type": "major"
```

Comment:
I'm modifying props of VersionTab so thought it might be breaking change at consumer end but it's not exported to be used there. Hence,it could be minor.